terraform/datadog/cloud_controller.tf

```diff
@@ -114,10 +114,10 @@ resource "datadog_monitor" "cc_job_queue_length" {
   escalation_message  = "Job queue in Cloud Controller API still too big, check the API health."
   require_full_window = false
 
-  query = "${format("max(last_30m):max:cf.cc.job_queue_length.total{deployment:%s} > 10", var.env)}"
+  query = "${format("avg(last_30m):max:cf.cc.job_queue_length.total{deployment:%s} > 10", var.env)}"
```

Comment:
30min with stuck CC seems like a very long time. I would like to be alerted before that if it happens. What about 10 min?